extensions/guacamole-auth-totp/src/main/java/org/apache/guacamole/auth/totp/user/TOTPUser.java

```diff
@@ -110,14 +116,16 @@ public void setAttributes(Map<String, String> attributes) {
         
         // Do not expose any TOTP secret attribute outside this extension
         attributes.remove(TOTP_KEY_SECRET_ATTRIBUTE_NAME);
-        
+        attributes.remove(TOTP_TIMEDRIFT_OFFSET_ATTRIBUTE_NAME);
+
         // Pull off the boolean reset field
         String reset = attributes.remove(TOTP_KEY_SECRET_RESET_FIELD);
         
         // If reset has been set to true, clear the secret.
         if (reset != null && reset.equals("true")) {
             attributes.put(TOTP_KEY_SECRET_ATTRIBUTE_NAME, null);
             attributes.put(TOTP_KEY_CONFIRMED_ATTRIBUTE_NAME, null);
+            attributes.put(TOTP_TIMEDRIFT_OFFSET_ATTRIBUTE_NAME, "0");
```

Comment:
I think in a reset case, this should be set to `null` and not `0`, as we want the rows to actually come out of the DB (which is what setting to `null` does).